workflow-service/src/main/java/com/redhat/parodos/workflow/definition/controller/WorkFlowDefinitionController.java

```diff
@@ -90,4 +100,18 @@ public ResponseEntity<WorkFlowDefinitionResponseDTO> getWorkFlowDefinitionById(@
 		return ResponseEntity.ok(response);
 	}
 
+	@Operation(summary = "Returns updated parameter value")
+	@ApiResponses(
+			value = {
+					@ApiResponse(responseCode = "200", description = "Succeeded",
+							content = { @Content(mediaType = "application/json",
+									schema = @Schema(implementation = List.class)) }),
+					@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content),
+					@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content) })
+	@PostMapping("/parameters/update/{valueProviderName}")
+	public ResponseEntity<List<WorkParameterValueResponseDTO>> updateParameter(@PathVariable String valueProviderName,
+			@RequestBody List<WorkParameterValueRequestDTO> workParameterValueRequestDTOS) {
+		return ResponseEntity.ok(workFlowParameterService.getValues(valueProviderName, workParameterValueRequestDTOS));
```

Comment:
Should we have the parameter a UUID for the workflow definition ID? So if the parameter is the same for the same WD is just weird

Reply:
yes,  it should an existing parameter from WD.   users' data providers can use the parameter-key to get the parameter data from db.  key should be unique for parameters within the same work.  UUID can also work.

Reply:
@eloycoto  Maybe we can also create a  **Parameter** table  and map it to workflowDefinition & workflowTaskDefinition table. so that records in Parameters can also have relations e.g. allof, anyof ....  And of course UUID will make more sense in this way